modules/sh-camera/meta.json

```diff
@@ -0,0 +1,6 @@
+{
+    "title": "SH Camera",
+    "description": "Sh Camera Module",
```

Comment:
Can we name the module just "Camera"?

Reply:
sure, my understanding was that there might be some other camera modules as well so wanted to keep it slightly unique.